magento/pwa-studio
# [doc] env variable tutorial

## Description

Creates a new tutorial on how to use environment variables in front end code.

## Related Issue

Closes PWA-798

## Acceptance

Any developer

### Verification Stakeholders

Any developer

### Specification



### Verification Steps

1. Navigate to the `pwa-devdocs` directory: `cd pwa-devdocs`
1. Start the html preview server: `yarn develop`
1. Navigate to the tutorials section
1. Verify **Add a new environment variable** is listed on the left nav
1. Verify links leads to expected page
1. Verify tutorial is accurate

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
